proxy/envoy/fault.go

```diff
@@ -40,14 +40,19 @@ func buildFaultFilters(routeConfig *HTTPRouteConfig) []HTTPFilter {
 
 // buildFaultFilter builds a single fault filter for envoy cluster
 func buildHTTPFaultFilter(cluster string, faultRule *proxyconfig.HTTPFaultInjection, headers Headers) *HTTPFilter {
+	abort, delay := buildAbortConfig(faultRule.Abort), buildDelayConfig(faultRule.Delay)
+	if abort == nil && delay == nil {
```

Comment:
No, it is valid if you have just 1 abort or 1 delay. The only invalid case is having neither.